ParProcCo/job_scheduler.py

```diff
@@ -127,38 +331,38 @@ def run(
         return self._run_and_monitor(job_indices)
 
     def _run_and_monitor(self, job_indices: List[int]) -> bool:
-        session = drmaa2.JobSession()  # Automatically destroyed when it is out of scope
-        self._run_jobs(session, job_indices)
-        self._wait_for_jobs(session)
+        self._run_jobs(job_indices)
+        self._wait_for_jobs()
         self._report_job_info()
         return self.get_success()
 
-    def _run_jobs(self, session: drmaa2.JobSession, job_indices: List[int]) -> None:
+    def _run_jobs(self, job_indices: List[int]) -> None:
```

Comment:
Best to rename and update this to reflect that jobs are being submitting (not run)